helper/testhelpers/testhelpers.go

```diff
@@ -952,17 +954,19 @@ func SetupLoginMFATOTP(t testing.T, client *api.Client) (*api.Client, string, st
 
 	// Create a test entity and alias
 	entityClient, entityID, _ := CreateEntityAndAlias(t, client, mountAccessor, "entity1", "testuser1")
+	methodName := "foo"
```

Comment:
So `SetupLoginMFATOTP` will always return "foo"? Should we make this a bit more flexible by allowing the caller to specify a methodName?